index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>InvertedIndex-App</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.js"></script>
+  <script src="./src/inverted-index.js"></script>
+  <script src="./src/app.js"></script>
+</head>
+
+<body ng-app='Index'>
+  <div ng-controller="IndexCtrl">
+    <nav class="navbar navbar-inverse">
+      <div class="container-fluid">
+        <div class="navbar-header">
+
+          <a class="navbar-brand" href="#">InvertedIndex</a>
+        </div>
+        <ul class="nav navbar-nav">
+          <!--TODO-->
+          <!--<li><a href="#">About</a></li>-->
+          <!--<li><a href="#">Tutorial</a></li>-->
+        </ul>
+
+        <!--TODO-->
+        <!--<ul class="nav navbar-nav navbar-right">
+        <li><a href="#"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+        <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+      </ul>-->
+        <form class="navbar-form navbar-right">
+          <div class="form-group">
+            <input type="text" class="form-control" placeholder="Search" ng-model="query">
+          </div>
+          <select id="drop-search" ng-model="fileSearch">
```

Comment:
Why not use bootstrap to it's fullest even for select box formatting at least for consistency